src/assets/tabs.js

```diff
@@ -53,7 +53,12 @@ function activateTab(hashEvent, eltID, scrollToElt) {
   if (eltID === undefined && window.location.hash)
     eltID = window.location.hash.slice(1);
 
-  var section = document.querySelector("section#" + eltID);
+  try {
+    var section = document.querySelector("section#" + eltID);
+  } catch {
+    section = null;
```

Comment:
A strict linter would reject this because in the event that `document.querySelector("section#" + eltID)` throws an error, `section` is undefined and undeclared when it is assigned.

Reply:
Yeah I'm eternally surprised that Javascript lets me get away with things like this.